tungsten_tempest_plugin/tests/api/contrail/test_routers.py

```diff
@@ -250,59 +224,66 @@ def test_delete_bgp_router(self):
     @rbac_rule_validation.action(service="Contrail",
                                  rules=["list_global_vrouter_configs"])
     @decorators.idempotent_id('4af768d1-3cbe-4aff-bcbc-0e045cac3277')
-    def test_list_global_vrouter(self):
+    def test_list_global_vrouter_configs(self):
         """test method for list global vrouter config objects"""
         with self.rbac_utils.override_role(self):
             self.router_client.list_global_vrouter_configs()
 
+    @decorators.skip_because(bug="1792446")
     @rbac_rule_validation.action(service="Contrail",
                                  rules=["create_global_vrouter_configs"])
     @decorators.idempotent_id('e13d800f-9304-4a06-9bf1-ad08345a13a8')
-    def test_create_global_vrouter(self):
+    def test_create_global_vrouter_configs(self):
         """test method for create global vrouter config objects"""
-        # Create global system config
-        global_system_config = self._create_global_system_config()['name']
+        # This test may make your environment unstable
+        # Juniper JTAC 2018-0912-0503
         with self.rbac_utils.override_role(self):
-            self._create_global_vrouter_config(global_system_config)
+            self._create_global_vrouter_config()
 
+    @decorators.skip_because(bug="1792446")
     @rbac_rule_validation.action(service="Contrail",
                                  rules=["show_global_vrouter_config"])
     @decorators.idempotent_id('3bb6f4e1-fd3f-4338-8392-f7f80974a80e')
     def test_show_global_vrouter_config(self):
         """test method for show global vrouter config objects"""
-        # Create global system config
-        global_system_config = self._create_global_system_config()['name']
-        global_vrouter_config_uuid = self._create_global_vrouter_config(
-            global_system_config)['uuid']
+        # This test may make your environment unstable
+        # Juniper JTAC 2018-0912-0503
```

Comment:
Are JTAC cases publicly searchable? I'm concerned about any comment about making an environment unstable, but particularly if the comment is supported by a reference that someone wouldn't easily be able to lookup. Is there a hyperlink that could be included here? Also, is there any warning elsewhere? Someone just cloning the repo and running the tests probably wouldn't see this comment before potentially damaging their environment.

Reply:
they are not accessible to everyone

Reply:
anyway this test will be skipped: @decorators.skip_because(bug="1792446")

Reply:
I hadn't noticed the skip_because, so I'm ok with including the test case now. However, I still think we should try to provide some publicly accessible explanation of a comment about making your environment unstable. Is there something in Launchpad that we can reference so that people who are not Juniper or AT&T can read if they want to understand more details about why this test is problematic?

Reply:
I don't have access to the bug on Launchpad, I will try to find anyone with access

Reply:
I added an explanation